### ISSUE TITLE
Press review presenter chooser

### ISSUE THREAD

**Issue by carlosaore** [carlosaore/student-tool#6]
Needs a section to randomly choose the next person to do the press review

**Comment by vmalep** [comment]
Good idea. It could also be an additional feature to the Stand up section, that the next on to speak is selected randomly.
